ADD_BLOG_ENTRY.md

```diff
@@ -0,0 +1,76 @@
+## Adding a blog entry
+
+Blog entries can be added by following these steps:
+
+- If submitting for the first time an [author](#blog-author) YAML file must be added to the `data/blog/authors` directory.
```

Comment:
Replaced with:
`- If submitting for the first time, an [author](#blog-author) YAML file must be added to the `data/blog/authors` directory.`
was missing a comma after 'first time'